assets/js/googlesitekit/components-gm2/Option.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Option component.
```

Comment:
We don't need this purely wrapper component and this will probably cause issues with `ref`. Instead, we should do what we did for the `Dialog` and it's associate components in the [components-gm2/index.js](https://github.com/google/site-kit-wp/blob/5a4f569143f54fc77117af6003839908ab84fab7/assets/js/googlesitekit/components-gm2/index.js#L19-L23).

Reply:
Updated, thanks!